### PR TITLE
Diff testing via API

### PR DESCRIPTION
**Pull request by amit1rrr**

### CODE REVIEW

**Comment by patcon** [comment]
Hi Amit! Cool tool! Thanks!

I'm not sure if you're aware, but its impossible to find examples of reviewnb working in the wild -- I've been hunting for 15 minutes, but just screenshots and examples of travesties from the old world of JSON diff's :)

**Comment by amit1rrr** [comment]
@patcon You can't find the examples because we only launched last week. You can install the app and try it yourself. [https://reviewnb.com](https://reviewnb.com)

If you just wish to see how the diff looks like there are a bunch of screenshots on our website (linked above).

File: notebooks/02.08-Sorting.ipynb [hunk 1]
```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: Intro is nice. Can we make the book logo centerd at the top? <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=0>ReviewNB</a>_

Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: Ever is an extra word in there. Can we remove it? <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=3>ReviewNB</a>_

Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: This is our first comment for a code cell. This better look good :) <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=4>ReviewNB</a>_

Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: Here is another comment for a code cell. This one has execute_result output type. <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=5>ReviewNB</a>_

Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: I don't like the shape of this output. Let's make it look like bow & arrow! <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=46>ReviewNB</a>_

File: notebooks/04.02-Simple-Scatter-Plots.ipynb [hunk 2]
```diff
@@ -37,10 +37,8 @@
   },
```
Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: You would think so, but these scatter plots are not really that simple. <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=2>ReviewNB</a>_

**Review by amit1rrr — COMMENTED** [review]
Review-1- All of these are fake comments for internal testing of ReviewNB! Do not be alarmed at the content, I'm better than this (or worse) :)

File: notebooks/04.02-Simple-Scatter-Plots.ipynb (hunk 2 above)
Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: Why talk so much? <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=21>ReviewNB</a>_

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: Some nice colorful output. Now that's what I call a scatter plot! <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=19>ReviewNB</a>_

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: Link it away! <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=22>ReviewNB</a>_

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit2rrr) [reply notebooks/02.08-Sorting.ipynb]: Why would I want to do that? **Instead checkout this statement that I have written in bold.** 

I care about following:

- A bullet list
- in the comment
- need to render properly

Also, checkout this hyperlinked [article](https://medium.com/) on medium.

@amit1rrr Now it's your turn to make sure at mentions and PR/issue references are rendering fine.  #3

> This is a quote

Following is a code snippet
```suggestion
   "cell_type": "markdown",
```

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: You have given a lot of different types of content to render. But don't worry, I got it under control. Here's some random text (source):

Now that the config is in `values.yaml` we are not using application flags for cert-manager.
I think it's a still a good idea to keep this functionality so apps in future can pass helm installation flags via this arg. Thoughts?

Comment (amit2rrr) [reply notebooks/02.08-Sorting.ipynb]: Pretty good. Now I will add to your woes. Here's a task list.

- [ ] Task 1 to be done

- [ ] Task 2 to be done

Comment (amit2rrr) [reply notebooks/02.08-Sorting.ipynb]: Yes. That's now taken care of.

Here's some longer blurb with a line break because I like long comments.

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: Thanks for fixing it. This conversation is now resolved.

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: "This is commenty from Postman for a newly create API"

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>a</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: This is a comment via API

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: "This is commenty from Postman for a newly create API"

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: "This is commenty from Postman for a newly create API"

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: "This is commenty from Postman for a newly create API"

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: "This is commenty from Postman for a newly create API"

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Let's post something.</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Let's post something.</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Post something 2.</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Post something 2.</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Post something 3</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Post something 4</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>Post <a href="https://reviewnb.com" target="_blank">something</a> 4</p><p><br></p><ul><li>Bullet 1</li><li>2</li><li>3</li></ul><p><br></p><p>I am <strong>bold</strong></p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>abcd</p>

**Comment by amit1rrr** [comment]
This is issue comment via API <p> is it? </p>

**Comment by amit1rrr** [comment]
This is a comment via API

**Comment by amit1rrr** [comment]
<p>      <span style="color: rgb(156, 220, 254);">owner_login:</span> <span style="color: rgb(206, 145, 120);">''</span>,</p><p>      <span style="color: rgb(156, 220, 254);">repo_name:</span> <span style="color: rgb(206, 145, 120);">''</span>,</p><p>      <span style="color: rgb(156, 220, 254);">pull_number:</span> <span style="color: rgb(206, 145, 120);">''</span>,</p>

**Comment by amit1rrr** [comment]
<p>This is a comment from <a href="https://reviewnb.com" target="_blank">ReviewNB</a> UI </p><p><br></p><ol><li>A</li><li>B</li><li>C</li></ol><p><br></p><p>Let's go <strong>bold</strong>.</p>

**Comment by amit1rrr** [comment]
<p>a</p>

**Comment by amit1rrr** [comment]
<p>b</p>

**Comment by amit1rrr** [comment]
<p>c</p>

**Comment by amit1rrr** [comment]
<p>d</p>

**Comment by amit1rrr** [comment]
<p>ab cd ef</p>

**Comment by amit1rrr** [comment]
<p>This is a <a href="http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/" target="_blank">self</a> link.</p><p><br></p><ol><li>A</li><li>B</li><li>C</li></ol><p><br></p><p>I am <em>italics</em></p>

**Comment by amit1rrr** [comment]
<p><span style="background-color: rgb(255, 255, 255);">&gt; I am&nbsp;</span><em style="background-color: rgb(255, 255, 255);">italics</em></p><p><br></p><p><span style="background-color: rgb(255, 255, 255);">Let's see how quote looks.</span></p>

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>a</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>b</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>ab</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>awe</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>aqe</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>aws</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>aqs</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>a</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>a</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>b</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>ab</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>This is a line</p><p>This is a new line</p><p><br></p><p>This is a new line after empty line above</p><p><br></p>

**Comment by amit1rrr** [comment]
<blockquote><span style="background-color: rgb(255, 255, 255);">Let's see how quote looks.</span></blockquote><p><span style="background-color: rgb(255, 255, 255);">Above quote does not seem useful.</span></p>

**Comment by amit1rrr** [comment]
<p>This is a new comment. </p><p><br></p><ul><li>1</li><li>2</li><li>3</li></ul>

**Comment by amit1rrr** [comment]
<p>when will this release?</p>

**Comment by amit1rrr** [comment]
<p>These 3 things remaining:</p><ul><li>A</li><li>B</li><li>C</li><li>D</li></ul>

**Review by amit1rrr — COMMENTED** [review]
Review-1- All of these are fake comments for internal testing of ReviewNB! Do not be alarmed at the content, I'm better than this (or worse) :)

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: Let's see if css class shows up. 

<p class='reviewnbTrailingText'>Reply via <a href='https://app.reviewnb.com' state='OPEN' cell_index=2>ReviewNB</a></p>

**Review by amit1rrr — COMMENTED** [review]
Review-1- All of these are fake comments for internal testing of ReviewNB! Do not be alarmed at the content, I'm better than this (or worse) :)

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: Let's see if div data & css class shows up. 

<p class='reviewnbTrailingText'>Reply via <a href='https://app.reviewnb.com' state='OPEN' cell_index=2>ReviewNB</a></p><div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Review by amit1rrr — COMMENTED** [review]
Review-1- All of these are fake comments for internal testing of ReviewNB! Do not be alarmed at the content, I'm better than this (or worse) :)

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: This is 1st comment via ReviewComment API

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: notebooks/04.02-Simple-Scatter-Plots.ipynb (hunk 2 above)
Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb:37]: <p>Very first comment.</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>Very first comment.</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>Very first comment.</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>Very first comment.</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb:30]: <p>Up to this point we have&nbsp;*<em>ever</em>*&nbsp;been concerned V7</p>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: notebooks/04.02-Simple-Scatter-Plots.ipynb (hunk 2 above)
Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb:37]: <p>Comment on the initial book image.</p><ol><li>B1</li><li>B2</li><li>B3</li></ol><p> </p><p>Let's find <a href="http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/files/" target="_blank">peace</a>.</p>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: <p>As any first-year computer comment.</p><p><br></p><ul><li>A</li><li>B</li><li>C</li><li>D</li></ul><p><br></p><p>Let's roll.</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: <p><strong>Silly sorting.</strong></p><p><br></p><pre class="ql-syntax" spellcheck="false">a = b + c
</pre><p><br></p><p>Is that right?</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

File: notebooks/04.02-Simple-Scatter-Plots.ipynb (hunk 2 above)
Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb:37]: <p>A Scatter Plots with&nbsp;<code>plt.plot </code></p>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb:37]: <p>1vsvs</p>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='5'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>fvds</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>A</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>Scatter plot 0, 10, 30</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit-nurtch) [inline notebooks/02.08-Sorting.ipynb]: <p>Comment from a new account!</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Comment by amit1rrr** [comment]
<p>ABC</p>

**Comment by amit1rrr** [comment]
<p>DEF</p>

**Comment by amit1rrr** [comment]
<p>GH</p>

**Comment by amit1rrr** [comment]
<p>A</p>

File: notebooks/04.02-Simple-Scatter-Plots.ipynb (hunk 2 above)
Comment (amit1rrr) [reply notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>A</p>

Comment (amit1rrr) [reply notebooks/04.02-Simple-Scatter-Plots.ipynb]: <p>B</p>

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>A</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>A</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>A</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>A</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>B</p>

Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <p>A</p>

**Comment by amit1rrr** [comment]
<blockquote>Comment from a new account!</blockquote>

**Comment by amit1rrr** [comment]
<blockquote>As any first-year computer science major will tell you, the selection sort is useful for its simplicity, but is much too slow to be useful for larger arrays. For a list of&nbsp;N&nbsp;values, it requires&nbsp;Nloops, each of which does on order&nbsp;\sim N&nbsp;comparisons to find the swap value. In terms of the "big-O" notation often used to characterize these algorithms (see&nbsp;<a href="http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/1/#Aside:-Big-O-Notation" target="_blank">Big-O Notation</a>), selection sort averages&nbsp;\mathcal{O}[N^2]: if you double the number of items in the list, the execution time will go up by about a factor of four.</blockquote><p><br></p><p>Is this accurate?</p>

File: notebooks/02.08-Sorting.ipynb (hunk 1 above)
Comment (amit1rrr) [reply notebooks/02.08-Sorting.ipynb]: <blockquote>ABC</blockquote><p>A </p>

Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: <blockquote>Up to this point</blockquote><p>alright!</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: notebooks/04.02-Simple-Scatter-Plots.ipynb (hunk 2 above)
Comment (amit1rrr) [inline notebooks/04.02-Simple-Scatter-Plots.ipynb]: <pre class="ql-syntax" spellcheck="false">a = b + C
</pre><p>Is this the addition you want?</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by amit1rrr** [comment]
<p>vf </p><p>Is this true?</p><p><br></p><p><code>serializer = CommitSerializer(instance=commit.fill_info())</code></p><p><br></p><p>Let's go!</p>

**Comment by amit1rrr** [comment]
<pre class="ql-syntax" spellcheck="false">            commit = Repository(repo_name, owner_login, user=user).get_commit(oid)
            serializer = CommitSerializer(instance=commit.fill_info())
            Audit.objects.create(user=user, api=Audit.COMMIT_INFO, owner_login=owner_login, repo_name=repo_name, commit_oid=oid)
            return Response(serializer.data)
</pre><p><br></p>

File: notebooks/02.08-Sorting.ipynb [hunk 3]
```diff
@@ -444,9 +420,7 @@
   {
    "cell_type": "code",
    "execution_count": 14,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
```
Comment (amit1rrr) [inline notebooks/02.08-Sorting.ipynb]: draft comment

**Pull request closed without merging**